src/pool/pool_scalable.c

```diff
@@ -62,11 +65,20 @@ static struct tbb_callbacks g_tbb_ops;
 static UTIL_ONCE_FLAG tbb_is_initialized = UTIL_ONCE_FLAG_INIT;
 static bool Init_tbb_global_state_failed;
 
+#ifdef _WIN32
```

Comment:
I would move this somewhere to utils and implement OS abstraction layer for such functionality.

Reply:
totally agree

Reply:
@vinser52 is it OK now?

Reply:
Fixed.
Done.